webpack.config.js

```diff
@@ -0,0 +1,52 @@
+const path = require('path');
+const webpack = require('webpack');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
```

Comment:
Not that it would make any difference, but just out of interest: When do you capitalize identifiers and when do you lowercase them? UpperCase if you instantiate them?

Reply:
I must admit I just copy-pasted this.. There is no logic here